pkg/print/print.go

```diff
@@ -56,10 +67,36 @@ func IsJSONLogEnabled() bool {
 	return logAsJSON
 }
 
+// StatusEvent reports a event log with given status.
+func StatusEvent(w io.Writer, status logStatus, fmtstr string, a ...any) {
+	if logAsJSON {
+		logJSON(w, string(status), fmt.Sprintf(fmtstr, a...))
+	} else {
+		if (w == os.Stdout || w == os.Stderr) && runtime.GOOS != windowsOS {
```

Comment:
Same here, we can invert this and return early.

Reply:
done